.github/workflows/build_installer.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish WRIMS GUI installer
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    permissions: write-all
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Build with Gradle
+        env:
+          PKG_USERNAME: ${{ github.actor }}
+          PKG_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./gradlew clean
+          ./gradlew :wrims-core:build
+          ./gradlew build
+          ./gradlew :wrims-gui:zipWrimsGui
+
+      - name: Attach WRIMS Installer zip to github deployment
+        run: |
+          gh release upload ${{ github.event.release.tag_name }} wrims-gui/build/installer/wrims_gui_x64_${{ github.event.release.tag_name }}.zip --repo ${{ github.repository }}
```

Comment:
Is there an advantage to publishing it this way vs a Gradle Publish step that publishes to the GH Repo via Maven compat? That would allow doing `publishToMavenLocal` for local testing too, which is a benefit of doing it via Gradle... Then the publish step is just `gradlew publish` or similar, maybe passing in the repo path via a Gradle prop

Reply:
This attaches it to a release vs just putting it in the package repo